rules/aws_cloudtrail_rules/aws_ec2_startup_script_change.yml

```diff
@@ -0,0 +1,71 @@
+AnalysisType: rule
+Description: 'Detects changes to the EC2 instance startup script. The shell script will be executed as root/SYSTEM every time the specific instances are booted up. '
+DisplayName: AWS EC2 Startup Script Change
+Enabled: false
+Filename: aws_ec2_startup_script_change.py
+Reports:
+    MITRE ATT&CK:
+        - T1059
+Severity: High
+Tests:
+    - ExpectedResult: true
+      Log:
+        awsregion: us-east-1
+        eventid: abc-123
+        eventname: ModifyInstanceAttribute
+        eventsource: ec2.amazonaws.com
+        eventtime: "2022-07-17 04:50:23"
+        eventtype: AwsApiCall
+        eventversion: "1.08"
+        p_any_aws_instance_ids:
+            - testinstanceid
+        p_event_time: "2022-07-17 04:50:23"
+        p_log_type: AWS.CloudTrail
+        p_parse_time: "2022-07-17 04:55:11.788"
+        requestParameters:
+            instanceId: testinstanceid
+            userData: <scripthere>
+      Name: ModifyInstanceAttributeUserdata
+    - ExpectedResult: false
+      Log:
+        awsregion: us-east-1
+        eventid: abc-123
+        eventname: ModifyInstanceAttribute
+        eventsource: ec2.amazonaws.com
+        eventtime: "2022-07-17 04:50:23"
+        eventtype: AwsApiCall
+        eventversion: "1.08"
+        p_any_aws_instance_ids:
+            - testinstanceid
+        p_event_time: "2022-07-17 04:50:23"
+        p_log_type: AWS.CloudTrail
+        p_parse_time: "2022-07-17 04:55:11.788"
+        requestParameters:
+            instanceId: testinstanceid
+            instanceType:
+                value: t3.nano
+      Name: ModifyInstanceAttribute-NoUserData
+    - ExpectedResult: false
+      Log:
+        awsregion: us-east-1
+        eventid: abc-123
+        eventname: ModifyImageAttribute
+        eventsource: ec2.amazonaws.com
+        eventtime: "2022-07-17 04:50:23"
+        eventtype: AwsApiCall
+        eventversion: "1.08"
+        p_any_aws_instance_ids:
+            - testinstanceid
+        p_event_time: "2022-07-17 04:50:23"
+        p_log_type: AWS.CloudTrail
+        p_parse_time: "2022-07-17 04:55:11.788"
+        requestParameters:
+            instanceId: testinstanceid
+            instanceType:
+                value: t3.nano
+      Name: NoModifyInstanceAttribute
+DedupPeriodMinutes: 60
```

Comment:
I suspect this doesn't matter to the computers involved, and convention indicates these items from 67-71 should appear before the test cases are defined.